test/test_torch.py

```diff
@@ -6002,6 +6002,59 @@ def test_conv_transposed_backward_agnostic_to_memory_format(self, device):
         input_ = layer_norm(input_.transpose(1, 2).contiguous()).contiguous()
         input_.sum().backward()
 
+    def test_future_div(self, device):
+        import torch
+
+        prev_div = torch.div
+        prev_tensor_div = torch.Tensor.div
+        prev_tensor_div_ = torch.Tensor.div_
+        prev_tensor_truediv = torch.Tensor.__truediv__
+
+        # Note: reload required since this test may be run multiple times
+        import torch.future_div
+        import importlib
+        importlib.reload(torch.future_div)
+
+        self.assertTrue(torch.div is torch.true_divide)
+        self.assertTrue(torch.Tensor.div is torch.Tensor.true_divide)
+        self.assertTrue(torch.Tensor.div_ is torch.Tensor.true_divide_)
+        self.assertTrue(torch.Tensor.__truediv__ is torch.Tensor.true_divide)
+
+        # Tests dense tensor behavior
+        t = torch.tensor((2,), device=device, dtype=torch.int)
+        d = torch.tensor((1,), device=device, dtype=torch.int)
+        o = torch.empty((2,), device=device, dtype=torch.int)
+        self.assertTrue((t / d).dtype, torch.get_default_dtype())
```

Comment:
I think you need to make this work with python 2.

Reply:
This test is gone so it's moot.